docker-compose.yml

```diff
@@ -1,9 +1,13 @@
 version: "3"
 
 services:
+  hugo_base:
+    build: ./
   hugo:
-    build: .
-    command: bash buildDeploy.sh server --buildDrafts
+    build:
+      context: ./
+      dockerfile: Dockerfile_hugo
+    command: tail -F anything
```

Comment:
I did this so the container would keep running. Is there a better way to do this?

Reply:
That's fine. People normally either do pings or tail something. I've typically seen it as `tail -f /dev/null`, but I fine with this. Maybe this is a manifestation of lack of knowledge of hugo, but if this container is serving the static hugo site, would there be an appropriate command that can be placed here to run the site and keep the container from shutting down. :thinking: